google/auth/iam.py

```diff
@@ -67,7 +82,7 @@ def _make_signing_request(self, message):
         message = _helpers.to_bytes(message)
 
         method = "POST"
-        url = _SIGN_BLOB_URI.format(self._service_account_email)
+        url = _IAM_SIGN_ENDPOINT.format(self._service_account_email) + "?alt=json"
```

Comment:
Was the missing `?alt=json` a bug?

Reply:
Shall we move the `?alt=json` part to the `_IAM_SIGN_ENDPOINT`